src/Shared/AWS/AWSSemanticConventions.cs

```diff
@@ -494,9 +488,6 @@ private AWSSemanticConventionsBase GetSemanticConventionVersion()
             case SemanticConventionVersion.V1_28_0:
                 return new AWSSemanticConventions_V1_28_0();
 
-            case SemanticConventionVersion.Legacy:
-                return new AWSSemanticConventionsLegacy();
```

Comment:
Shouldn't the implementation class AWSSemanticConventionsLegacy also be removed? Is it still needed somewhere?

Reply:
The AWSSemenaticConventions classes use inheritance so they only need to capture the diff between versions.  So 1.28 inherits from Legacy and 1.29 inherits from 1.28.

```csharp
private class AWSSemanticConventions_V1_28_0 : AWSSemanticConventionsLegacy
```
and 
```csharp
private class AWSSemanticConventions_V1_29_0 : AWSSemanticConventions_V1_28_0
```

But I did mark `AWSSemanticConventionsLegacy` as `asbtract` to further highlight that it shouldn't be used directly.